README.md

```diff
@@ -23,6 +23,10 @@ Logging Service to demonstrate the distributed logging with log collector.
       git clone https://github.com/sumedhak27/Containerized-Distributed-Logger.git
       cd Containerized-Distributed-Logger
       ```
+    - Build docker image of logger application
+      ```bash
+      docker build -f Dockerfile -t demo-logger:1.1 .
```

Comment:
we should keep the <version_num> instead of 1.1

Reply:
line number 51 https://github.com/sumedhak27/Containerized-Distributed-Logger/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R51
needs a change 
`less /var/log/demoLogger/[container_id],log`